util/dvsim/Deploy.py

```diff
@@ -283,6 +287,16 @@ def get_timeout_mins(self):
         """Returns the timeout in minutes."""
         return 0
 
+    def extract_runtimes(self, log_text: List):
+        """Input the logfile in lines and extract the job_runtime."""
+        try:
+            self.job_runtime.time, self.job_runtime.unit = get_job_runtime(
+                log_text, self.sim_cfg.tool)
+        except NotImplementedError:
+            log.debug("Using dvsim default runtime due to unsupported tool: " + self.sim_cfg.tool)
+        except SyntaxError:
+            log.debug("Parsing job runtime has syntax error with " + self.sim_cfg.tool)
```

Comment:
Sorry the conversation get lost here:
https://github.com/lowRISC/opentitan/pull/13793#discussion_r933629643